src/ios.rs

```diff
@@ -52,11 +61,18 @@ impl CredentialApi for IosCredential {
     /// Returns a [NoEntry](ErrorCode::NoEntry) error if there is no
     /// credential in the store.
     fn get_password(&self) -> Result<String> {
-        let password_bytes =
-            get_generic_password(&self.service, &self.account).map_err(decode_error)?;
+        let password_bytes = self.get_secret()?;
         decode_password(password_bytes.to_vec())
```

Comment:
Please remove this call, as it's unnecessary.  `password_bytes` is already a Vec<u8>.

Reply:
Agreed, thats done.